File: dashboard-service/src/main/java/com/franrl/dashboard_service/dto/FeedbackClassificationNotification.java
package com.franrl.dashboard_service.dto;

import com.franrl.enums.Sentiment;
import com.franrl.enums.UrgencyLevel;

import java.util.UUID;

public record FeedbackClassificationNotification(
        UUID feedbackId,
        UrgencyLevel urgency,
        Sentiment sentiment,
        String category,
        boolean needsReview
) {
}

File: feedback-triage-frontend/src/app/core/models/FeedbackClassificationNotification.ts
import { Sentiment } from '../enums/Sentiment';
import { UrgencyLevel } from '../enums/UrgencyLevel';

export interface FeedbackClassificationNotification {
  feedbackId: string;
  urgency: UrgencyLevel;
  sentiment: Sentiment;
  category: string;
  needsReview: boolean;
}